onnx_tf/handlers/backend/sequence_length.py

```diff
@@ -0,0 +1,15 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("SequenceLength")
+class SequenceLength(BackendHandler):
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    input_sequence = tensor_dict[node.inputs[0]]
+
+    return [tf.shape(input_sequence.to_sparse())[0]]
```

Comment:
spec says this should return `int64` but `tf.shape` defaults to `int32`

Reply:
i.e. `tf.shape(input_sequence.to_sparse(), out_type=tf.int64)`

Reply:
Yeah, fixed. Need to change the master branch too.